tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -17,7 +17,7 @@
     DEFAULT_MODEL_CLASSES,
     IMAGE_GEN_CLS
 )
-import openvino.runtime.opset13 as opset
+import openvino.opset13 as opset
```

Comment:
please, add try / except for compatibility with older versions

and merge with latest master

Reply:
Done